### PR TITLE
ROC-6827: Added amount to defendant settled dashboard

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-6827

### Change description
Added amount to defendant settled dashboard

### Developer self-QA run statement

- [ *] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [* ] No

### CODE REVIEW

File: src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
 {% macro paidInFullDefendantClaimDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ claimantName }} confirmed you paid on {{ moneyReceivedOn }}.',
-      { claimantName: claim.claimData.claimant.name, moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+  <p>{{ t('{{ claimantName }} confirmed you paid {{ totalAmountTillToday }} on {{ moneyReceivedOn }}.',
+      { claimantName: claim.claimData.claimant.name,
+        totalAmountTillToday: claim.totalAmountTillToday | numeral,
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk]: This value will change day to day if interest is included. we need to either save the value at the time the case is settled or change the content

Comment (KiranVarma02) [reply src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk]: content agreed with business and updated. we are not showing amount for now. I believe we are raising bug later - @nsonje is handling

File: src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 {% macro paidInFullDefendantClaimDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ claimantName }} confirmed you paid on {{ moneyReceivedOn }}.',
+  <p>{{ t('{{ claimantName }} confirmed you settled on {{ moneyReceivedOn }}.',
       { claimantName: claim.claimData.claimant.name, moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
   <p>{{ downloadResponseLink(claim) }}</p>
 {% endmacro %}
```
Comment (robertparkinson) [inline src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk]: as discussed, need the other two statuses updated as they are currently wrong in production and it makes sense to fix them and keep them all the same.

Comment (KiranVarma02) [reply src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk]: done

**Pull request merged**